packages/stark-ui/src/modules/dropdown/components/dropdown.component.ts

```diff
@@ -123,6 +140,10 @@ export class StarkDropdownComponent implements OnInit, OnChanges, OnInit {
 		this.logger.debug(componentName + ": component initialized");
 		this.optionsAreSimpleTypes = this.areSimpleTypes();
 		this.setDefaultBlank();
+		if (this.color) {
+			const nativeElement: HTMLElement = this.elementRef.nativeElement;
+			nativeElement.classList.add("stark-" + this.color);
```

Comment:
I don't think we should add the `stark-`prefix to the color, otherwise in case the developer wants to use his own color (i.e.   red), he should add a CSS rule for the `stark-red` but in fact it is not an stark color.... why not simply calling the colors like `color-white`, `color-red` and so on? Or even better to name them `color-theme-white`, `color-theme-red` so it is clear that those are linked to the `$color-theme` map. What do you think?

Reply:
Thinking a bit more on this, couldn't we abstract this new property in an abstract class that we could extend from? This way it will be easier to apply the same functionality in other stark components...

Reply:
About the first remark, here is our new stark color map:

```
$color-theme: (
  accent: (
    color: #8f0039,
    contrast: $light-primary-text
  ),
  alert: (
    color: #d32f2f,
    contrast: $light-primary-text
  ),
  alt: (
    color: #323232,
    contrast: $light-primary-text
  ),
  neutral: (
    color: #bfbfbf,
    contrast: $light-primary-text
  ),
  primary: (
    color: #0076c8,
    contrast: $light-primary-text
  ),
  success: (
    color: #4caf50,
    contrast: $light-primary-text
  ),
  warning: (
    color: #ff9800,
    contrast: $light-primary-text
  ),
  white-transp: (
    color: rgba(255, 255, 255, 0.87),
    contrast: $dark-primary-text
  )
);
```
And the idea was to call the color stark-accent, stark-primary etc... so to me it is quite clear this way. If you check the `_base.css` in stark-ui, all the stark class have stark as a prefix, so to me it make more sense keeping it this way for every provided classes.

@SuperITMan any opinion on this?